source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -299,28 +295,24 @@ package:
        and/or deserialize a property.
 
    * - ``BsonProperty``
-     - Specifies a custom document field name when converting the POJO
-       field to BSON. You can include a discriminator to serialize POJOs
-       nested within the field.
-
-       .. important::
+     - | Specifies a custom document field name when converting the POJO
+         field to BSON. You can include a discriminator to serialize POJOs
+         nested within the field. 
+       | 
+       | When applying ``@BsonProperty`` to a private field, 
+         **you must also add getter and setter methods for that field to serialize and 
+         customize the field name**.
 
-          When applying ``@BsonProperty`` to a private field, you must also add
-          getter and setter methods for that field to serialize and customize
-          the field name.
 
    * - ``BsonExtraElements``
-     - Specifies the POJO field on which to deserialize all elements that are
-       not mapped to a field. The POJO field must be one of the following
-       types:
-
-       - `Document <{+api+}/apidocs/bson/org/bson/Document.html>`__
-       - `BsonDocument <{+api+}/apidocs/bson/org/bson/BsonDocument.html>`__
-       - ``Map<String, Object>``
-
-       .. seealso::
-
-          :ref:`BsonExtraElements Annotation Example <bsonextraelements-annotation-code-example>`
+     - | Specifies the POJO field on which to deserialize all elements that are
+         not mapped to a field. The POJO field must be one of the following
+         types:
+         - `Document <{+api+}/apidocs/bson/org/bson/Document.html>`__
+         - `BsonDocument <{+api+}/apidocs/bson/org/bson/BsonDocument.html>`__
+         - ``Map<String, Object>``
+       |  
```

Comment:
```suggestion
         types:
         - `Document <{+api+}/apidocs/bson/org/bson/Document.html>`__
         
         - `BsonDocument <{+api+}/apidocs/bson/org/bson/BsonDocument.html>`__
         
         - ``Map<String, Object>``
       |  
```
The spacing is a bit off for this list in staging: https://deploy-preview-581--docs-java.netlify.app/fundamentals/data-formats/pojo-customization/#bsonrepresentation-error-example.

These spaces might fix.

Reply:
Fixed!